### PR TITLE
suppressing by id

### PR DESCRIPTION
**Pull request by marwin1991**
Implement: https://github.com/angrymeir/SecScanner2JUnit/issues/19#issuecomment-2480998445

### CODE REVIEW

**Comment by marwin1991** [comment]
cc @frakman1

**Comment by marwin1991** [comment]
@angrymeir can I ask you for fast code review and merge and release for @frakman1

 :)

**Comment by marwin1991** [comment]
@angrymeir friendly reminder

**Comment by angrymeir** [comment]
@marwin1991 Thank you so much for the implementation! Looks good to me :) I'll do one final test run tonight and then we're ready to go 🚀

File: secscanner2junit/config.py [hunk 1]
```diff
@@ -2,15 +2,18 @@
 
 
 class Suppression:
-    def __init__(self, type, value):
+    def __init__(self, id, type, value):
```
[the remark this answers is not in the document]
Comment (angrymeir) [reply secscanner2junit/config.py:5]: Please let me know if that makes sense to you :)

Btw: In the next days I'll integrate the unit tests into our CI, so we can automatically see if there's any issues

[the remark this answers is not in the document]
Comment (angrymeir) [reply secscanner2junit/config.py:5]: I guess in addition we'll have to adjust 
`expected = Config([Suppression('cwe', '2555'), Suppression('find_sec_bugs_type', 'SPRING_ENDPOINT')])`

to 

`expected = Config([Suppression(None, 'cwe', '2555'), Suppression(None, 'find_sec_bugs_type', 'SPRING_ENDPOINT')])`

in  `/tests/test_config.py`

**Review by angrymeir — APPROVED** [review]
Thank you so much @marwin1991!
I'll push the new release once you merged it :)

**Comment by marwin1991** [comment]
@angrymeir let's go 🚀 🌔

**Pull request merged**